### DISCUSSION TITLE
Choices between 2 types of cycle lane

### DISCUSSION THREAD

**Discussion by Kazing** [streetcomplete/StreetComplete#3963] [Q&A]
Hello,
In the quest : "Is there a cycleway here? What type?"
I am confused between two choices for cycle lane : 
- "cycle lane"
- "cycle lane (delimited by paint)"

![Screenshot_20220413-223648](https://user-images.githubusercontent.com/41507326/163267251-c3343d59-4677-4d85-a26f-8146ad3e5bb3.png)

In France cycle lane are delimited by paint, so I don't see difference between the 2 choices and the drawings doesn't help me
I came across another (french) user confused by this two type of cycle lane : https://www.openstreetmap.org/note/3132564

What can be described with "cycle lane (delimited by paint)" any example ?

Thanks :)

**Comment by FloEdelmann** [comment]
It was discussed and changed in #3461. Maybe that helps answer the question?

**Reply by Kazing** [reply]
Thank you,
After reading #3461, I understand that "cycle lane (delimited by paint)" add the tag `cycleway=lane` + `cycleway:lane=advisory` 
This can describe what is called Chaucidou or Chaussée à voie centrale banalisée in France.
The drawing is clear but the wording (at least in French) is confusing. I understand the french wording shall not be exclusive to France, so "Chaucidou" is not appropriate.

I will ask french community for their views, in order to improve this wording.

**Reply by FloEdelmann** [reply]
If a better wording is found, this can be added only in the french translation. However, it should also work for French users in other countries.

**Reply by matkoniecz** [reply]
See https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app for how to improve translation

**Comment by Binnette** [comment]
Hi @Kazing from France 🇫🇷 , I propose the following: "accotement (empruntable par véhicules à moteur)".

Source: https://www.fub.fr/velo-ville/amenagements/amenagements-types/accotements-cyclables-campagne-dorenavant-ville

![image](https://user-images.githubusercontent.com/6597721/175021763-8b4f0da9-ac96-466f-a525-bd0014921783.png)

**Reply by matkoniecz** [reply]
@Binnette See https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app - and feel free to translate it better! Sadly I am unable to judge this translation (or even retype it with French variant of ą and ę)

**Reply by westnordost** [reply]
It was discussed (by Frenchmen) in https://github.com/streetcomplete/StreetComplete/issues/3461 though
